Firestore/core/test/firebase/firestore/local/memory_local_store_test.cc

```diff
@@ -26,25 +29,48 @@ namespace {
 
 class TestHelper : public LocalStoreTestHelper {
  public:
+  TestHelper(std::unique_ptr<QueryEngine> query_engine, bool index_free)
```

Comment:
Same comments as in the other one (the bool can go away but making a QueryEngine::type).

Reply:
Removed.